styles/styles.css

```diff
@@ -0,0 +1,155 @@
+header {
+  background-image: url('../assets/hero.jpg');
+  /*background-position: 50% 50%;*/
+  background-size: cover;
+  background-repeat: no-repeat;
+  border-bottom: solid;
+  border-color: #9D9494;
+  height: 100px;
+  color: #93737F;
+  font-family: 'Raleway', sans-serif;
+  text-align: center;
+  margin-top: 0;
+  padding: 100px;
+}
+
+header h1 {
+  margin-top: 15px;
+  font-size: 3em;
+}
+
+
+header nav {
+  padding: 0;
+  margin-top: 100px;
+}
+
+header nav li {
+  display: inline-block;
+  margin-left: 8px;
+  margin-right: 40px;
+}
+
+header nav li a {
+  color: #93737F;
+  text-decoration: none;
+  font-size: 1.3em;
+}
+
+a:hover {
+  color: black;
+}
+
+h2 {
+  text-align: center;
+}
+
+.circular--landscape {
+  display: inline-block;
+  position: relative;
+  width: 400px;
+  height: 400px;
+  overflow: hidden;
+  border-radius: 50%;
+}
+
+.portfolio nav li {
+  display: inline-block;
+}
+
+.portfolio li {
+  padding-left: 3%;
+}
+
+.portfolio ul li {
+  display: inline-block;
+  padding: 2%;
+  margin-left: 10%;
+}
+
+.portfolio ul li:nth-child(2) {
+  margin-left: 23%;
+  margin-right: 12%;
+}
+
+li {
+  list-style-type: none;
+}
+
+a {
+  color: #93737F;
+  text-decoration: none;
+}
+
+body {
+  width: 100%;
+  font-family: 'Raleway', sans-serif;
+  color: #93737F;
+}
+
+#welcome {
+  clear: both
+}
+
+#welcome h2 {
+  width: 40%;
+  padding-top: 1%;
+  margin: 0;
+  font-family: 'Raleway', sans-serif;
+  font-size: 2em;
+  text-align: center;
+}
+
+#welcome p {
+  width: 40%;
+  float: left;
+  margin-left: 4%;
+  font-size: 1.5em;
+}
+
+#welcome img {
+  width: 40%;
+  float: right;
+  margin-right: 4%;
+}
+
+#about {
+  clear: both
+}
+
+#about h2 {
+  width: 40%;
+  padding-top: 1%;
+  margin-left: 60%;
+  font-family: 'Raleway', sans-serif;
+  font-size: 2em;
+  text-align: center;
+}
+
+#about p {
+  width: 40%;
+  float: right;
+  margin-right: 4%;
+  font-size: 1.5em;
+}
+
+#about img {
+  width: 40%;
+  float: left;
```

Comment:
This places it against the left side, you might consider throwing a more general margin around the whole thing if you want it pushed away from the side a bit like you have on the homepage 😃